skeleton/Rakefile

```diff
@@ -1,5 +1,6 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet/vendor/semantic/lib/semantic'
+require 'puppet/version'
+require 'puppet/vendor/semantic/lib/semantic' unless Puppet.version.to_f < 3.6
```

Comment:
Isn't this the wrong way round?

"metadata-json-lint has a dependency on semantic; which isn't in Puppet versions lower than 3.6." I read this as "install semantic on versions of puppet < 3.6" ie. the opposite to what the patch does.

Or am I just missing something?